### ISSUE TITLE
Crash when doing /reload

### ISSUE THREAD

**Issue by wodim** [AeroScripts/QuestieDev#1622]
Happened once. Next /reload it didn't happen. Maybe a race condition?

```
Message: ...ddOns\Questie\Modules\FramePool\QuestieFramePool.lua:319: attempt to index local 'iconFrame' (a nil value)
Time: Wed Dec 11 13:20:09 2019
Count: 1
Stack: ...ddOns\Questie\Modules\FramePool\QuestieFramePool.lua:319: attempt to index local 'iconFrame' (a nil value)
...ddOns\Questie\Modules\FramePool\QuestieFramePool.lua:319: in function `CreateLine'
...ddOns\Questie\Modules\FramePool\QuestieFramePool.lua:275: in function `CreateWaypoints'
...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:1344: in function `DrawAvailableQuest'
...erface\AddOns\Questie\Modules\Quest\QuestieQuest.lua:1372: in function `DrawAllAvailableQuests'
...rface\AddOns\Questie\Modules\QuestieEventHandler.lua:230: in function <...rface\AddOns\Questie\Modules\QuestieEventHandler.lua:224>
[C]: ?
...sic\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:29: in function <...sic\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:25>
...sic\Libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:64: in function `Fire'
...ce\AddOns\Questie\Libs\AceEvent-3.0\AceEvent-3.0.lua:120: in function <...ce\AddOns\Questie\Libs\AceEvent-3.0\AceEvent-3.0.lua:119>

Locals: <none>
```

**Comment by BreakBB** [comment]
@wodim We need to know which version of Questie you're using.

**Comment by wodim** [comment]
v5.2.3

**Comment by BreakBB** [comment]
We released v5.3.0 yesterday and will release another update in an hour or so. This might have been fixed.
